README.md

```diff
@@ -1,5 +1,7 @@
 # knife-supermarket
 
+This Plugin is DEPRECATED: the code has been moved into core Chef.
```

Comment:
Maybe include more information in the README dep notice. Suggested: "the feature has been moved into core Chef in versions greater than 12.11.18 and no longer requires a separate gem installation."